### PR TITLE
fix dark theme by removing custom css

### PR DESCRIPTION
**Pull request by jamestalton**
Signed-off-by: James Talton <jtalton@redhat.com>

### CODE REVIEW

File: src/AcmLabels/AcmLabels.tsx [hunk 1]
```diff
@@ -54,40 +49,20 @@ export function AcmLabels(props: {
...
     return (
-        <span className="acm-labels">
+        <LabelGroup numLabels={labels.length} expandedText={expandedText} collapsedText={collapsedText}>
             {labels.map((label) => (
-                <Tooltip key={label} content={label}>
-                    <span className="acm-label">{label}</span>
-                </Tooltip>
+                <Label key={label}>
+                    <Truncate content={label} />
+                </Label>
```
Comment (KevinFCormier) [inline src/AcmLabels/AcmLabels.tsx:59]: @jamestalton Great to use components, but Truncate is in beta and for some reason it sets a minimum width.
image

image

Seems strange that a component that aims to save space would use a minimum amount. Do you think we should override that variable and set it to 0? Or we can ask why the mid width for Truncate.

Comment (jamestalton) [reply src/AcmLabels/AcmLabels.tsx:59]: Had to use the Truncate to solve the problem in ACM details pages where labels overflowed the details column.

Comment (jamestalton) [reply src/AcmLabels/AcmLabels.tsx:59]: Screen Shot 2022-06-09 at 12 34 43 PM

Comment (jamestalton) [reply src/AcmLabels/AcmLabels.tsx:59]: The minimum of 0 would work.
There is an internal span that has a second minimum that we cannot access easily.

Comment (jamestalton) [reply src/AcmLabels/AcmLabels.tsx:59]: Screen Shot 2022-06-09 at 12 40 16 PM

**Pull request merged**